core/src/main/kotlin/org/evomaster/core/problem/external/service/httpws/ExternalServiceHandler.kt

```diff
@@ -200,22 +220,19 @@ class ExternalServiceHandler {
         wm.start()
 
         // to prevent from the 404 when no matching stub below stub is added
-        // TODO: Need to decide what should be the default behaviour
+        // WireMock throws an exception when there is no stub for the request
+        // to avoid the exception it handled manually
         wm.stubFor(
             any(anyUrl())
                 .atPriority(10)
                 .willReturn(
                     aResponse()
-                        .withStatus(500)
-                        .withBody("Internal Server Error")
+                        .withStatus(404)
```

Comment:
are the `404` and `Not Found` default settings of wiremock instance? then it might better to define it as private constant variable.